.eslintrc

```diff
@@ -40,6 +40,7 @@
     "no-debugger": 1,
     "no-var": 1,
     "import/no-unresolved": 0,
+    "import/no-extraneous-dependencies": 0,
```

Comment:
I needed to add this because `npm-link` does not add the package to package.json